Src/FluentAssertions/Primitives/ReferenceTypeAssertions.cs

```diff
@@ -416,6 +417,44 @@ public AndConstraint<TAssertions> NotBeAssignableTo(Type type, string because =
         return new AndConstraint<TAssertions>((TAssertions)this);
     }
 
+    /// <summary>
+    /// Asserts that the criteria provided by the element inspector is satisfied.
+    /// </summary>
+    /// <param name="expected">The element inspector which must be satisfied by the <typeparamref name="TSubject" />.</param>
```

Comment:
🤔 Same here. We're not satisfying an inspector but executing one or more assertions.

Reply:
You have not addressed this one either?

Reply:
I must be confused. I thought both comments were addressed by your suggestion? Honestly, I'm not used to working with GitHub code reviews, so I'm sorry if I'm a bit slow (I'm using Azure DevOps daily, so it's a very different way of working 🙂)